index.html

```diff
@@ -1,12 +1,374 @@
+                совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его
+                на ваш этаж.
+              </p>
+            </li>
+            <li class="service-item-desk">
+              <h3 class="service-title">Гарантия</h3>
+              <p class="service-link">
+                Мы с удовольствием доставим ваш товар прямо к вашему подъезду
+                совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его
+                на ваш этаж.
+              </p>
+            </li>
+            <li class="service-item-desk">
+              <h3 class="service-title">Кредит</h3>
+              <p class="service-link">
+                Мы с удовольствием доставим ваш товар прямо к вашему подъезду
+                совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его
+                на ваш этаж.
+              </p>
+            </li>
+          </ul>
+        </section>
+        <section class="order-delivery">
+          <h2 class="visualy-hidden">Доставка</h2>
+          <div class="order-delivery-inner">
+            <div class="decor"></div>
+            <strong>Доставим редкий товар под заказ</strong>
+            <div class="order-delivery-info"></div>
+          </div>
+        </section>
+        <section class="details">
+          <div class="about">
+            <h2 class="about-title">О нас</h2>
+            <p class="about-text">
+              Огромный выбор гаджетов не оставит равнодушным гика, который есть
+              в каждом из нас.
+            </p>
+            <p class="about-text">
+              Мы можем доставить ваш товар в самые отдалённые уголки России
+              транспортными компаниями:
+            </p>
+            <ul class="delivery-company-list">
+              <li class="delivery-company-item">
+                <h3 class="about-link">Меловые линии</h3>
+              </li>
+              <li class="delivery-company-item">
+                <h3 class="about-link">СДИК</h3>
+              </li>
+              <li class="delivery-company-item">
+                <h3 class="about-link">ТоксБерри</h3>
+              </li>
+            </ul>
+            <a class="detail-link button" href="#">Подробнее о нас</a>
+          </div>
+          <div class="contacts">
+            <h2 class="about-title">Контакты</h2>
+            <p class="about-text">
+              Вы можете забрать товар сами, заехав в наш офис. Заодно сможете
+              проверить работоспособность покупки.
+            </p>
+            <h3 class="contacts-subtitle">Интернет-магазин «Девайс»</h3>
+            <address class="contacts-adress">
+              <p class="about-text">
+                Санкт-Петербург, набережная реки Карповки, 5, литера П.
+              </p>
+              <a
+                class="contacts-addres_link-phone botton"
+                href="tel:78128121212"
+                >+7 (812) 812-12-12</a
+              >
```

Comment:
Вот эти моменты нужно везде исправить. Если хочешь атрибуты написать в несколько строк, то давай так

```suggestion
              <a
                  class="contacts-addres_link-phone botton"
                  href="tel:78128121212"
              >
                  +7 (812) 812-12-12
              </a>
```